scheds/rust/scx_layered/src/main.rs

```diff
@@ -2282,6 +2292,26 @@ impl<'a> Scheduler<'a> {
         Ok(())
     }
 
+    fn refresh_idle_qos(&mut self) -> Result<()> {
+        if !self.idle_qos_enabled {
+            return Ok(());
+        }
+
+        for layer in self.layers.iter() {
+            let idle_resume_us = layer.kind.common().idle_resume_us as i32;
```

Comment:
I'm not sure if this proper handles CPUs that aren't in any layer.

Reply:
Yeah, if there's no open layer, `available_cpus()` wouldn't be represented anywhere. Make a map of all cpus initizlied to 0 and then populate it while iterating layers and then apply the result?